src/c++/perf_analyzer/genai-pa/genai_pa/wrapper.py

```diff
@@ -52,9 +52,18 @@ def run(model, args=None):
                     cmd += f"-b {value} "
                 else:
                     if len(arg) == 1:
-                        cmd += f"-{arg} {value}"
+                        cmd += f"-{arg} {value} "
                     else:
                         arg = utils.convert_option_name(arg)
                         cmd += f"--{arg} {value} "
+        # TODO: Once the OpenAI endpoint support is in place in PA core,
+        # update the input-data option arg
+        # cmd += f"--input-data {DEFAULT_INPUT_DATA_JSON}  -p 10000 -s 99"
+        cmd += f"--input-data ./input_data.json -p 10000 -s 99"
```

Comment:
You could also uncomment this code now, and just change `DEFAULT_INPUT_DATA_JSON=input_data.json` until the other filename workflow is ready

Reply:
I will handle this in the next PR is that is okay with you.